src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -5,6 +5,8 @@ mappings:
                 type: 'keyword'
             family:
                 type: 'keyword'
+            enabled:
```

Comment:
Yes, with @SamirBoulil we think that for attributes it's better to be exhaustive